bridge-deploy/bridge_deploy/home.py

```diff
@@ -0,0 +1,118 @@
+import json
+from typing import Dict, NamedTuple
+from web3.contract import Contract
+from deploy_tools.deploy import (
+    deploy_compiled_contract,
+    increase_transaction_options_nonce,
+    send_function_call_transaction,
+)
+
+
+class DeployedHomeBridgeResult(NamedTuple):
+    home_bridge: Contract
+    home_bridge_block_number: int
+
+
+def load_contract(contract_name):
+    with open(
+        f"../poa-bridge-contracts/build/contracts/{contract_name}.json"
+    ) as json_file:
+        return json.load(json_file)
+
+
+def deploy_home_bridge_contract(
+    *, web3, transaction_options: Dict = None, private_key=None
+):
+
+    if transaction_options is None:
+        transaction_options = {}
+
+    latest_block = web3.eth.getBlock("latest")
+
+    # Deploy home bridge proxy
+    eternal_storage_proxy_src = load_contract("EternalStorageProxy")
+    home_bridge_storage_contract: Contract = deploy_compiled_contract(
```

Comment:
why do we need the type hint here? Is it missing in the deploy tools?

Reply:
Honestly: Copy-Pasted from Auction-Deploy. Seems not to be required. Removed.